src/ReplicatedStorage/Common/Config/FallDamage.lua

```diff
@@ -1,6 +1,6 @@
 local FallDamageConfig = {
-    Threshold = 60;
-    Scale = 1;
+    Threshold = 150;
+    Scale = 1.7;
```

Comment:
Why are these called Threshold and Scale? I'm not really sure what these are, without reading the code.